src/django_components/util/testing.py

```diff
@@ -376,16 +376,6 @@ def wrapper_outer(*args: Any, **kwargs: Any) -> Any:  # type: ignore[misc]
                 param_names, values = parametrize
                 ids = None
 
-            for value in values:
```

Comment:
I came across this bit from the implementation of `djc_test` decorator. But this check is no longer needed.